react/javascript/src/components/gherkin/MDG.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import ReactMarkdown from 'react-markdown'
+import rehypePlugins from '../app/rehypePlugins'
+import GherkinQueryContext from '../../GherkinQueryContext'
+import Step from './Step'
+import styles from '../app/GherkinDocumentList.module.scss'
```

Comment:
I think we should rename these styles to `./Gherkin.module.css`

Reply:
Feels a little broad. Is just `DocumentList` maybe good enough?